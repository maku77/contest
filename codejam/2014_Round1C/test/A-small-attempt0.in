100
343/695
1/4
1/4
433/590
239/512
251/333
83/256
15/128
1/64
5/16
43/64
1/16
9/128
111/128
29/128
179/512
1/256
1/4
1/44
5/64
1/512
1/64
7/512
1/16
316/329
3/16
1/8
1/128
5/128
11/64
81/256
7/16
5/8
177/211
7/256
79/195
3/256
269/976
1/32
1/8
1/32
1/128
1/8
1/256
1/8
275/977
139/242
1/4
61/64
1/64
9/512
5/16
23/64
5/256
31/32
91/128
1/8
1/256
784/985
1/128
40/147
121/256
7/512
1/16
1/64
279/512
98/423
463/741
1/2
7/16
1/16
9/64
1/256
15/16
1/4
21/256
99/256
13/32
15/16
1/16
1/32
399/920
1/16
15/64
3/4
29/801
39/64
1/256
1/2
1/32
342/949
3/8
3/64
57/64
107/128
161/421
3/128
3/7
180/457
1/4
