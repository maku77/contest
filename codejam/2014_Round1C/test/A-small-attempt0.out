Case #1: impossible
Case #2: 2
Case #3: 2
Case #4: 1
Case #5: 2
Case #6: impossible
Case #7: 2
Case #8: 4
Case #9: 6
Case #10: 2
Case #11: 1
Case #12: 4
Case #13: 4
Case #14: 1
Case #15: 3
Case #16: 2
Case #17: 8
Case #18: 2
Case #19: 6
Case #20: 4
Case #21: 9
Case #22: 6
Case #23: 7
Case #24: 4
Case #25: impossible
Case #26: 3
Case #27: 3
Case #28: 7
Case #29: 5
Case #30: 3
Case #31: 2
Case #32: 2
Case #33: 1
Case #34: impossible
Case #35: 6
Case #36: impossible
Case #37: 7
Case #38: 2
Case #39: 5
Case #40: 3
Case #41: 5
Case #42: 7
Case #43: 3
Case #44: 8
Case #45: 3
Case #46: impossible
Case #47: 1
Case #48: 2
Case #49: 1
Case #50: 6
Case #51: 6
Case #52: 2
Case #53: 2
Case #54: 6
Case #55: 1
Case #56: 1
Case #57: 3
Case #58: 8
Case #59: impossible
Case #60: 7
Case #61: impossible
Case #62: 2
Case #63: 7
Case #64: 4
Case #65: 6
Case #66: 1
Case #67: impossible
Case #68: impossible
Case #69: 1
Case #70: 2
Case #71: 4
Case #72: 3
Case #73: 8
Case #74: 1
Case #75: 2
Case #76: 4
Case #77: 2
Case #78: 2
Case #79: 1
Case #80: 4
Case #81: 5
Case #82: 2
Case #83: 4
Case #84: 3
Case #85: 1
Case #86: impossible
Case #87: 1
Case #88: 8
Case #89: 1
Case #90: 5
Case #91: impossible
Case #92: 2
Case #93: 5
Case #94: 1
Case #95: 1
Case #96: impossible
Case #97: 6
Case #98: impossible
Case #99: impossible
Case #100: 2
